src/ToricVarieties/NormalToricVarieties/attributes.jl

```diff
@@ -80,7 +77,7 @@ An error is triggered if it is not yet set.
 """
 function coefficient_ring(v::AbstractNormalToricVariety)
     if !has_attribute(v, :coefficient_ring)
-        error("Coefficient ring not yet set.")
+        return QQ
     end
     return get_attribute(v, :coefficient_ring)
 end
```

Comment:
I would suggest to also set this attribute if you want to return the default value. The same applies to `coordinate_names`.